alvr/server/src/connection.rs

```diff
@@ -481,6 +481,12 @@ fn connection_pipeline(
         warn!("Chosen refresh rate not supported. Using {fps}Hz");
     }
 
+    let enable_foveated_encoding =
+        settings.video.foveated_encoding.enabled() && streaming_caps.supports_foveated_encoding;
+    if settings.video.foveated_encoding.enabled() && !streaming_caps.supports_foveated_encoding {
+        warn!("Foveated encoding is not supported by the client.");
```

Comment:
So are we permanently disabling ffe?. I think I did not make myself clear in my request. FFE has issues on many smartphones, but some smartphones still work with FFE and it significantly reduces the client side latency by about 50% (which is huge) 

So we have to by default disable it, but if user wants to force enable it it should be enabled. 

So imo, we have to detect two event i.e., when first installed, or user clicked "reset" in settings. Only then client request to disable FFE by default should be honored, rest if user goes to settings and manually enables it, then immaterial of client request, setting should be overridden by user choice.

Reply:
I'm not sure what you are asking is feasible. ALVR relies on alvr_session to provide the settings default, and client and server each link to it and have a separate copy. The client cannot change the default value of the server. The only way i see this working is if the client owns the ground truth for settings, which is also an imperfect solution.
In this case, this could be solved if you somehow can detect if the smartphone supports FFR or not, and then negotiate the value with the server.

Reply:
I don't want to have client side user settings. If it's not possible to detect those events maybe we can give a smol "force" checkbox in gui beside the setting, with a tag "Will force this setting on client even if client may not support it"

Reply:
Maybe that could work. It may be confusing for people on other headsets but it causes no harm. Best would be if you cal also report FFE support on an allowlist basis (could be a filter by android version or single model names).